adal/src/main/java/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -70,7 +70,7 @@
     /**
      * Default constructor for cache item.
      */
-    public TokenCacheItem() {}
+    public TokenCacheItem() { }
```

Comment:
>{ } [](start = 28, length = 3)

looks odd, make it like
public TokenCacheItem() {
    // Intentionally left blank
}